packages/core/src/modules/credentials/protocol/v1/__tests__/V1CredentialServiceProposeOffer.test.ts

```diff
@@ -1,6 +1,5 @@
 import type { CredentialStateChangedEvent } from '../../../CredentialEvents'
```

Comment:
I guess this file and other ex-service tests should be renamed to V1Credential**Protocol**XXX.test.ts